jobs/grafana/templates/credentials

```diff
@@ -0,0 +1,4 @@
+[default]
+aws_access_key_id = <%= p("grafana.aws.aws_access_key_id") %>
```

Comment:
Ah these should also have a default or something; right now this template will blow up rendering if those properties aren't specified.